local-testing/index.html

```diff
@@ -71,6 +71,11 @@ <h1>Amazon Connect - Custom Implementation of Customer Chat</h1>
                         contactAttributes: JSON.stringify({
                             "customerName": customerName
                         }),
+                        featurePermissions: {
+                            "ATTACHMENTS": false,  // this is the override flag from user for attachments
+                            "MESSAGING_MARKDOWN": true, // enable rich messaging
```

Comment:
Do we need both the flag in featurePermissions and the supportedMessagingContentTypes to enable this feature here? Can we only use one of them - maybe only the supportedMessagingContentTypes.
The ATTACHMENTS flag here is used as an override because the enabling attachment is done via AWS console

Reply:
agreed, removed in latest revision